doc/nrf/ug_thread_configuring.rst

```diff
@@ -498,19 +503,21 @@ This command builds two versions of the libraries, with and without debug symbol
 |board_note_for_updating_libs|
 The :kconfig:option:`CONFIG_OPENTHREAD_BUILD_OUTPUT_STRIPPED` Kconfig option will be disabled again after this command completes.
 
-Updating libraries to debug version
------------------------------------
+Updating the libraries with debug symbols
+-----------------------------------------
 
-You can also install the debug version of the current OpenThread libraries (from Zephyr).
-This can be useful when debugging, but will take a significant amount of storage memory.
-Take this into account while committing those libraries to the repository.
+There is a single command to update the libraries with debug symbols.
+Using the command line, run the command in the project folder.
```

Comment:
Good point. I originally had them as "If using..." but took those out since it sounded like there was a third option of using neither. "When using..." does not have that problem.